portal-web/test/functional/com/liferay/portalweb/paths/portlet/pageaudit/PageAudit.path

```diff
@@ -0,0 +1,34 @@
+<html>
+<head>
+<title>PageAudit</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+<thead>
+<tr><td rowspan="1" colspan="3">PageAudit</td></tr>
+</thead>
+<tbody>
+<tr>
+	<td>CHECK_ISSUES_TEXT</td>
+	<td>//*[contains(@class, 'font-weight-semi-bold') and contains(@class, 'c-mb-2')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CONNECT_TO_GOOGLE_PAGESPEED_BUTTON</td>
+	<td>//*[contains(@class, 'btn-secondary') and contains(text(), 'Connect To Google PageSpeed')]</td>
```

Comment:
if you run your test on a non-english page, this test will pass?
can you change this xpath to a more "generic stuff?"

Reply:
Hmm good idea, I'll modify it.